tools/deploy_s3.py

```diff
@@ -137,6 +137,8 @@ def deploy_to_s3_main(
                 # However, JsDelivr will currently not serve .ts file in the
                 # custom CDN configuration, so it does not really matter.
                 content_type = "text/x.typescript"
+            elif file_path.suffix == ".metadata":
```

Comment:
Maybe,
```suggestion
            elif file_path.name.endswith('.whl.metadata'):
```
to be a bit more specific.